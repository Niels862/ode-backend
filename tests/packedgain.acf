chip {
    cabs: [
        cab 1 with clocks 1, - {
            cams: [
                SumInv as sum1 {
                    inputs: 2
                }
            ]
        },
        cab 4 with clocks 1, - {
            cams: [
                GainInv as gain1 {},
                GainInv as gain2 {}
            ]
        }
    ],
    routing: [
        gain1 -> sum1:1,
        gain2 -> sum1:2
    ]
}
