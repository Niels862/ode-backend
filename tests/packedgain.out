0 0 0 0 32 4 0 2 5 0 0 64 0 0 81 255 15 241 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 64 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 2 255 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 255 255 255 255 255 255 0 0 0 0 0 0 0 0 0 0 0 0 1 19 1 129 1 28 1 129 1 29 1 129 12 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 48 0 16 0 0 0 192 0 16 0 5 0 208 0 16 0 0 255 255 255 255 255 255 0 0 0 0 0 0 0 0 0 0 0 0 1 19 1 129 1 30 1 129 1 31 1 129 12 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 48 0 16 0 0 0 224 0 16 0 5 0 240 0 16 0 0 255 255 255 255 255 255 0 0 0 0 0 0 0 0 0 0 0 0 1 19 1 129 1 30 1 129 1 31 1 129 12 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 48 0 16 0 0 0 224 0 16 0 5 0 240 0 16 255 255 255 255 255 255 255 255 0 0 0 0 0 0 0 0 1 18 1 130 1 16 1 130 1 19 1 129 1 16 1 129 12 0 0 0 0 0 0 0 0 0 0 0 0 32 0 32 0 0 0 32 0 5 0 48 0 16 0 5 0 0 0 16